timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/TimeLockTestUtils.java

```diff
@@ -36,27 +41,41 @@ private TimeLockTestUtils() {
         // Utility class
     }
 
+    private static final SslConfiguration SSL_CONFIGURATION =
+            SslConfiguration.of(Paths.get("var/security/trustStore.jks"));
+
     static TransactionManager createTransactionManager(TestableTimelockCluster cluster) {
         List<String> serverUris = cluster.servers().stream()
                 .map(server -> server.serverHolder().getTimelockUri())
                 .collect(Collectors.toList());
+
         AtlasDbConfig config = ImmutableAtlasDbConfig.builder()
                 .namespace("test")
                 .keyValueService(new InMemoryAtlasDbConfig())
-                .timelock(ImmutableTimeLockClientConfig.builder()
-                        .serversList(ImmutableServerListConfig.builder()
-                                .servers(serverUris)
-                                .sslConfiguration(SslConfiguration.of(Paths.get("var/security/trustStore.jks")))
-                                .build())
-                        .build())
                 .build();
         return TransactionManagers.builder()
                 .config(config)
                 .userAgent("test")
                 .globalMetricsRegistry(new MetricRegistry())
                 .globalTaggedMetricRegistry(DefaultTaggedMetricRegistry.getDefault())
+                .runtimeConfigSupplier(getAtlasdbRuntimeConfigSupplier(serverUris))
                 .build()
                 .serializable();
     }
 
+    private static ServerListConfig getServerListConfig(List<String> serverUris) {
+        return ImmutableServerListConfig.builder()
+                .servers(serverUris)
+                .sslConfiguration(SSL_CONFIGURATION)
+                .build();
+    }
+
+    private static Supplier<Optional<AtlasDbRuntimeConfig>> getAtlasdbRuntimeConfigSupplier(List<String> serverUris) {
```

Comment:
nit: `AtlasDbRuntimeConfig`

Reply:
nit: would prefer to have `getServerListConfig` below this method, for better top-down readability.